### PR TITLE
feat(logger): prefix log env settings with `RENOVATE_`

### PR DESCRIPTION
**Pull request by viceice**
## Changes
Prefix all logger relevant settings with `RENOVATE_`. Fallback to env vars without prefix.

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [ ] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [x] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: docs/usage/examples/self-hosting.md [hunk 1]
```diff
@@ -383,12 +383,12 @@ spec:
 
 ## Logging
 
-If you're ingesting/parsing logs into another system then we recommend you set `LOG_LEVEL=debug` and `LOG_FORMAT=json` in your environment variables.
+If you're ingesting/parsing logs into another system then we recommend you set `RENOVATE_LEVEL=debug` and `RENOVATE_LOG_FORMAT=json` in your environment variables.
```
Comment (rarkins) [inline docs/usage/examples/self-hosting.md:386]:
```suggestion
If you're ingesting/parsing logs into another system then we recommend you set `RENOVATE_LOG_LEVEL=debug` and `RENOVATE_LOG_FORMAT=json` in your environment variables.
```

**Review by rarkins — CHANGES_REQUESTED** [review]

File: lib/logger/index.ts [hunk 2]
```diff
@@ -123,15 +128,16 @@ loggerLevels.forEach((loggerLevel) => {
 });
 
 // istanbul ignore if: not easily testable
```
Comment (viceice) [inline lib/logger/index.ts:130]:
```suggestion
```

File: lib/logger/index.ts [hunk 3]
```diff
@@ -123,15 +128,16 @@ loggerLevels.forEach((loggerLevel) => {
...
-if (is.string(process.env.LOG_FILE)) {
+const logFile = getEnv('LOG_FILE');
+if (is.string(logFile)) {
```
Comment (viceice) [inline lib/logger/index.ts:132]:
```suggestion
// istanbul ignore if: not easily testable
if (is.string(logFile)) {
```

**Comment by viceice** [comment]
I've left some dev settings as `LOG_LEVEL`, so containerbase can benefit from it too

File: .github/label-actions.yml [hunk 4]
```diff
@@ -145,7 +145,7 @@
     A maintainer wants you to find and share a specific log message (`packageFiles with updates`) from your repository or environment.
 
 
-    If you self-host Renovate: make sure you run Renovate with `LOG_LEVEL=debug` to get the debug log messages! Next, open the debug-level logs and search for `packageFiles with updates`. This text marks the start of a structured log message that shows every package file, dependency, and update that Renovate found.
+    If you self-host Renovate: make sure you run Renovate with `RENOVATE_LOG_LEVEL=debug` to get the debug log messages! Next, open the debug-level logs and search for `packageFiles with updates`. This text marks the start of a structured log message that shows every package file, dependency, and update that Renovate found.
```
Comment (rarkins) [inline .github/label-actions.yml:148]: Do we need to add a note that LOG_LEVEL may be needed for older versions? Or just hope there isn't too much confusion during the changeover?

Comment (rarkins) [reply .github/label-actions.yml:148]: True, we can leave the old way, as it's still supported

**Review by viceice — COMMENTED** [review]
@rarkins should we defer all docs changes to not confuse with older versions?

**Review by rarkins — APPROVED** [review]

**Pull request merged**